tests/test_linkextractors.py

```diff
@@ -506,3 +509,32 @@ def test_link_restrict_text(self):
     @pytest.mark.xfail
     def test_restrict_xpaths_with_html_entities(self):
         super(LxmlLinkExtractorTestCase, self).test_restrict_xpaths_with_html_entities()
+
+    def test_filteringlinkextractor_deprecation_warning(self):
+        """Make sure the FilteringLinkExtractor deprecation warning is not
+        issued for LxmlLinkExtractor"""
+        with catch_warnings(record=True) as warnings:
+            LxmlLinkExtractor()
+            self.assertEqual(len(warnings), 0)
+
+            class SubclassedItem(LxmlLinkExtractor):
```

Comment:
What do you think about just "Subclassed", or "SubclassedLxmlLinkExtractor", or "SubclassedLinkExtractor"?